src/System.Management.Automation/engine/ExecutionContext.cs

```diff
@@ -478,6 +478,11 @@ internal LocationGlobber LocationGlobber
         /// </summary>
         internal Dictionary<string, Assembly> AssemblyCache { get; private set; }
 
+        /// <summary>
+        /// Ignores script requirements that fail.
+        /// </summary>
+        internal bool IgnoreFailedScriptRequirements { get; set; }
```

Comment:
This is an important internal structure, and I cannot accept adding a new property to it for the sake of a rare scenario in an interactive session.
I don't think it's worth it.
If you do not agree, then we must wait for the conclusion of the working group.

Reply:
I think the reason why I went with a property was because otherwise I'd have to update a bunch of methods with an extra parameter. I'd have to take another look to know if I still think it's the best way all these years later, but what's the harm in adding an internal property? It's hard to imagine how it would break anything by simply being there.

Reply:
> what's the harm in adding an internal property?

It is out of completion code. I can't make a personal decision without the approval of MSFT team.
And as I said, this change is too much edge to make many changes in code. It's better to spend time on something more useful.